tests/io/test_xlsx_table.py

```diff
@@ -0,0 +1,109 @@
+import os
+import pandas as pd
+import pytest
+
+from janitor import io
+from pandas.testing import assert_frame_equal
+
+
+TEST_DATA_DIR = "tests/test_data"
```

Comment:
Can we make use of Pathlib to provide a _resolved_ path?

E.g. `(here() / tests/test_data).resolve()`?

Doing so will make explicit to someone reading the code how the `test_data` dir is resolved, at least relative to the root dir (by use of `here()`), while the `.resolve()` will help with debugging the path (in case the need ever arises).

Reply:
what is `here()`? seems like an R thing. Do we have something similar in pyjanitor?

Reply:
Oh my, my bad - `here()` comes from `pyprojroot`, a project by @chendaniely. It's become so ingrained in my headspace that I forgot that it's actually an external dependency. I think we can ignore the request to use `here()`, though I think the `.resolve()` will help debug in the future.